listeners.py

```diff
@@ -33,13 +34,19 @@ def process_timeout(self, delta_seconds):
 class StatisticsWatcher(BaseWatcher):
     def __init__(self, get_logfile):
         self.stats = dict()
+        self.rss_results = sampler.rss_summary()
```

Comment:
Here we lean on the fact that the dictionary returned at `StatisticsWatcher` construction will never be replaced by another one. However it would be better to just pass a sampler object here, save it and call `self._sampler.rss_summary()` at the point, where we'll need the statistics (in `print_statistics()`).